### PR TITLE
rename migrations file for windows users

### PR DESCRIPTION
**Pull request by benjaminstrasser**
some :heart: for windows users

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		"test:integration": "cross-env DATABASE_LOCATION=:memory: vitest run --project integration",
 		"test:e2e": "playwright test",
 		"---- DB ------------------------------------------------------------": "",
-		"setupdb": "pnpm migrate && pnpm db-types",
+		"setupdb": "pnpm migrate:latest && pnpm db-types",
```
Comment (sjaghori) [inline package.json:20]: how is this a windows only issue? :D

Comment (sjaghori) [reply package.json:20]: also I think we should rename this step to e.g. `sync:db` and use it in the `pnpm dev` script.

Comment (benjaminstrasser) [reply package.json]: > how is this a windows only issue? :D

It is not I just changed it as I realized setupdb does not function properly.




> also I think we should rename this step to e.g. `sync:db` and use it in the `pnpm dev` script.

I agree the naming is not ideal.
How about `setup:db` for migration and sync and `sync:db` for type generation, which is similar to `sync:svelte`?

Comment (sjaghori) [reply package.json]: yes it's better.

**Review by sjaghori — APPROVED** [review]

**Pull request merged**